docs/index.asciidoc

```diff
@@ -39,6 +39,8 @@ include::./source-maps.asciidoc[]
 
 include::./distributed-tracing.asciidoc[]
 
+include::./message-queues.asciidoc[]
+
 include::./performance-tuning.asciidoc[]
```

Comment:
There should be an addition to supported-technologies.asciidoc for SQS now.

Reply:
_nod_ added a 

```
|https://www.npmjs.com/package/aws-sdk[aws-sdk] |>1 <3 |Will instrument SQS send/receive/delete messages
```

to `supported-technologies.asciidoc`.